parsers/shared/src/test/scala/sigmastate/lang/SigmaParserTest.scala

```diff
@@ -561,24 +551,6 @@ class SigmaParserTest extends AnyPropSpec with ScalaCheckPropertyChecks with Mat
     parseType("Coll[(Coll[Byte], (Coll[Long], Long))]") shouldBe SCollection(STuple(SByteArray, STuple(SLongArray, SLong)))
   }
 
-  property("negative tests") {
-    fail("(10", 1, 4)
-    fail("10)", 1, 3)
-    fail("X)", 1, 2)
-    fail("(X", 1, 3)
-    fail("{ X", 1, 4)
-    fail("{ val X", 1, 8)
-    fail("\"str", 1, 5)
-  }
```

Comment:
Looks like moving tests out of here is not motivated enough. You can re-implement the `fail` method without relying on SigmaCompiler, but using SigmaParser directly. If you look at the `parse` method, you will see it just delegates to SigmaParser.

Reply:
Yeah, right. Thanks for catching, I've updated the PR.